src/stdpar/Makefile

```diff
@@ -20,8 +20,15 @@ EXE_OBJ := $(patsubst $(SRC_DIR)%,$(OBJ_DIR)%,$(EXE_SRC:%=%.o))
 EXE_DEP := $(EXE_OBJ:$.o=$.d)
 
 LIBNAMES := $(filter-out plugin-% bin test Makefile% plugins.txt%,$(wildcard *))
-PLUGINNAMES := $(patsubst plugin-%,%,$(filter plugin-%,$(wildcard *)))
-MY_CXXFLAGS := -I$(TARGET_DIR) -DLIB_DIR=$(LIB_DIR)/$(TARGET_NAME)
+
+# PLUGINNAMES are compiled with nvcc, PLUGINNAMES_NVCXX with nvc++
+# PLUGINNAMES_NVCXX := SiPixelRecHits
+# PLUGINNAMES := $(filter-out $(PLUGINNAMES_NVCXX),$(patsubst plugin-%,%,$(filter plugin-%,$(wildcard *))))
+
+PLUGINNAMES_NVCXX := $(patsubst plugin-%,%,$(filter plugin-%,$(wildcard *)))
+PLUGINNAMES := 
```

Comment:
Do you see a need to keep the `PLUGINNAMES` around?

In principle I'd like to try out at some point what happens if one tries to limit the `nvc++` to the minimum (i.e. code with memory management or device code) and use `gcc` for the rest, but I think such a test would be better done later, and the build rules may be better to figure out then rather than thinking about them prematurely.

Reply:
`PLUGINNAMES` can be used to specifiy plugins that we want to compile with `gcc`. Since all of them can be compiled with `nvc++` I can remove it. 

`nvc++` has some restriction as to which code can be compiled with which compiler. Any code that does heap memory allocation that is to be used in a stdpar algorithm will need to be compiled by `nvc++`so that it can instrument the code to use unified memory.